libr/core/config.c

```diff
@@ -1533,9 +1543,11 @@ R_API int r_core_config_init(RCore *core) {
 	SETPREF("bin.classes", "true", "Load classes from rbin on startup");
 
 	/* cfg */
-	r_config_set_cb (cfg, "cfg.bigendian", CFG_BIGENDIAN, &cb_bigendian);
+	r_config_set_cb (cfg, "cfg.bigendian", CFG_BIGENDIANHOST, &cb_bigendian_host);
+	r_config_set_cb (cfg, "cfg.bigendiantarget", CFG_BIGENDIANTARGET, &cb_bigendian_target);
 	SETPREF("cfg.plugins", "true", "Load plugins at startup");
-	r_config_desc (cfg, "cfg.bigendian", "Use little (false) or big (true) endiannes");
+	r_config_desc (cfg, "cfg.bigendian", "Use little (false) or big (true) endianness for display");
+	r_config_desc (cfg, "cfg.bigendiantarget", "Use little (false) or big (true) endianness for asm/anal");
```

Comment:
why this needs to be different? there's only one endian to obey. the host one is fixed and know at compile time. the one in cfg.bigendian should be big or little, i dont see why display must be different than asm/anal. handling endianness is sometimes mad. so adding another var makes it even harder to maintain